examples/asr/speech_to_text_eval.py

```diff
@@ -154,6 +177,52 @@ def main(cfg: EvaluationConfig):
             f"contain value for `pred_text`."
         )
 
+    if cfg.use_per:
+        per_data_obj = PERData(
+            hypotheses=predicted_text,
+            references=ground_truth_text,
+            punctuation_marks=list(cfg.text_processing.punctuation_marks),
+        )
+        per_data_obj.compute()
+        per = per_data_obj.per
+
+    if cfg.scores_per_sample:
```

Comment:
Could you move this to a function inside of NeMo so that all this is hidden ? Maybe in transcribe utils ?

Reply:
Sure, fixed with the commit [dfc809a](https://github.com/NVIDIA/NeMo/pull/7538/commits/dfc809a2a891e96070a69c9a93d34d2f29cd90f2)